### DISCUSSION TITLE
How can I use MBS?

### DISCUSSION THREAD

**Discussion by LaraTheBunny** [bananarama92/MBS#95] [Q&A] [answered]
So idk if I'm just being stupid, but I was wondering how I could enter the fortune wheel and so  on... I was trying the whole day to find a way but to no avail :( 

I hope someone could help me out with this :3

**Comment by bananarama92** [comment, marked as answer]
For future reference: you can either type the `/wheel` command in chat if you just want to open your own wheel, or:
1. Make sure that the person whose wheel you're interested in equips the wheel of fortune item
2. Click on them and navigate to "Character Actions" -> "Spin the Wheel of Fortune"

![Spiiiiiiiiiiiiiin](https://github.com/bananarama92/MBS/assets/105885570/a695ba7f-dc1f-4e1a-bb8b-c2ac5340cbdb)
